### PR TITLE
hg-serve: add page

### PR DESCRIPTION
**Pull request by pxgamer**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/hg-serve.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- Start a web server instance with a specific name:
+
+`hg serve --name {{name}}`
+
+- Start a web server instance using the specified theme:
+
+`hg serve --style {{style}}`
+
+- Start a web server instance using the specified SSL certificate:
```
Comment (agnivade) [inline pages/common/hg-serve.md]: Where would one specify the ssl key ?

File: pages/common/hg-serve.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Start a web server instance on the specified listening address:
+
+`hg serve --address {{address}}`
+
+- Start a web server instance with a specific name:
+
+`hg serve --name {{name}}`
+
+- Start a web server instance using the specified theme:
```
Comment (agnivade) [inline pages/common/hg-serve.md]: Is there a predefined list of themes one can use ? Where do I get the list of themes ?

File: pages/common/hg-serve.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- Start a web server instance on the specified port:
+
+`hg serve --port {{port}}`
+
+- Start a web server instance on the specified listening address:
+
+`hg serve --address {{address}}`
+
+- Start a web server instance with a specific name:
```
Comment (agnivade) [inline pages/common/hg-serve.md]: What does it mean for a web server to have a name ?

Comment (pxgamer) [reply pages/common/hg-serve.md]: The name is what appears in the title of the pages. And also it is shown in tabs.

I wasn't quite sure of a good way to shorten it.

File: pages/common/hg-serve.md (hunk 2 above)
Comment (pxgamer) [reply pages/common/hg-serve.md]: It is predefined, but you can also specify a custom theme if it's added to the template directory.

The downside is that there is no way (as far as I know) to list themes, other than to do an `ls` in the themes directory.

Comment (agnivade) [reply pages/common/hg-serve.md]: Ok, then perhaps we can mention that in brackets.

File: pages/common/hg-serve.md (hunk 3 above)
Comment (agnivade) [reply pages/common/hg-serve.md]: How about "identifier" or "label" ?

File: pages/common/hg-serve.md (hunk 1 above)
Comment (pxgamer) [reply pages/common/hg-serve.md]: It looks like it needs to be a bundled `.PEM` file containing both the key and the certificate.

**Pull request merged**